File: examples/helloworld/stream/go/streamsAreCurrentlyTemporarilyNotSupportedInClients.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/helloworld"
	"os"
)

// Stream returns a stream of "Hello $name" responses
func StreamsAreCurrentlyTemporarilyNotSupportedInClients() {
	helloworldService := helloworld.NewHelloworldService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := helloworldService.Stream(&helloworld.StreamRequest{
		Name: "not supported",
	})
	fmt.Println(rsp, err)
}

File: examples/js/helloworld/stream/streamsAreCurrentlyTemporarilyNotSupportedInClients.js
const { HelloworldService } = require("m3o/helloworld");

// Stream returns a stream of "Hello $name" responses
async function streamsAreCurrentlyTemporarilyNotSupportedInClients() {
  let helloworldService = new HelloworldService(process.env.MICRO_API_TOKEN);
  let rsp = await helloworldService.stream({
    name: "not supported",
  });
  console.log(rsp);
}

streamsAreCurrentlyTemporarilyNotSupportedInClients();
